src/actions/public/L2/getProveWithdrawalTransactionArgs.test.ts

```diff
@@ -58,4 +61,11 @@ test('correctly generates args', async () => {
   // rome-ignore lint: ok unused variable
   const { withdrawalHash, ...withdrawalTransaction } = withdrawalMessages.messages[0]
   expect(args.withdrawalTransaction).toEqual(withdrawalTransaction)
+
+  const hash = await writeProveWithdrawalTransaction(walletClient, {
```

Comment:
Verified this test was broken as code as. Note this wasn't really an onchain issue, it would have been ok and we matched the SDK. But viem was throwing error.